src/shared/components/RadioButton/RadioButton.tsx

```diff
@@ -1,25 +1,41 @@
 import React from 'react'
-import { Label, Input, StyledInput, StyledLabelText, RadioButtonStyleProps } from './RadioButton.style'
+import {
+  Label,
+  Input,
+  StyledInput,
+  StyledLabelText,
+  StyledCaptionText,
+  RadioButtonStyleProps,
+  RadioButtonWrapper,
+} from './RadioButton.style'
 
 type RadioButtonProps = Partial<{
   selectedValue: string | number
   label: string
+  caption?: string
 }> &
   Omit<RadioButtonStyleProps, 'clickable'> &
   React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement> &
   Omit<React.InputHTMLAttributes<HTMLInputElement>, 'onClick'>
 
 const RadioButton = React.forwardRef<HTMLInputElement, RadioButtonProps>(
-  ({ value, selectedValue, label, disabled, error, ...props }, ref) => {
+  ({ value, selectedValue, label, caption, disabled, error, ...props }, ref) => {
     const isSelected = value === selectedValue
 
     return (
-      <Label disabled={disabled}>
-        <StyledInput checked={isSelected} error={error} disabled={disabled}>
-          <Input ref={ref} value={value} type="radio" disabled={disabled} {...props} checked={isSelected} />
-        </StyledInput>
-        {label && <StyledLabelText>{label}</StyledLabelText>}
-      </Label>
+      <RadioButtonWrapper>
+        <Label disabled={disabled}>
+          <StyledInput checked={isSelected} error={error} disabled={disabled}>
+            <Input ref={ref} value={value} type="radio" disabled={disabled} {...props} checked={isSelected} />
+          </StyledInput>
+          {label && <StyledLabelText variant="body1">{label}</StyledLabelText>}
+        </Label>
+        {caption && (
+          <StyledCaptionText variant="caption" error={error}>
```

Comment:
This is wrong. If you set `variant` to `caption` the element will be set as `<caption>` and you shouldn't use the `caption` element inside `div`. Captions, if I'm not mistaken, are related only to tables.
If you use this, you'll see in the console:
![image](https://user-images.githubusercontent.com/51168865/113702227-8518d980-96d9-11eb-8e0e-56d84ee3c651.png)

We should update `Text` component maybe in the future. This `caption` variant is very misleading

Reply:
Okay, leave it as it is. I will update the `Text` component in my PR #424